### PR TITLE
May Release highlights for JS

### PR DESCRIPTION
**Pull request by KarishmaGhiya**
- Added May Release highlights for JS

### CODE REVIEW

**Review by czubair — APPROVED** [review]

File: releases/2021-05/index.md [hunk 1]
```diff
@@ -11,9 +11,11 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
 
 ## Release Highlights
 
-*
-*
-*
+- SDK for JS has released new low level client packages for these services - Azure Purview Scanning, Azure Purview Catalog, Azure Confidential Ledger and Azure Document Translator
+- SDK for JS has released a new library for Azure Web PubSub service and another library specifically for event handlers for Express
+- Azure Communication Phone Numbers SDK for JS released GA for the first time, with version 1.0.0
+- Azure EventGrid SDK for JS has released new GA version. It has added new System Events: "Microsoft.PolicyInsights.PolicyStateCreated", "Microsoft.PolicyInsights.PolicyStateChanged", "Microsoft.PolicyInsights.PolicyStateDeleted", "Microsoft.Storage.AsyncOperationInitiated", "Microsoft.Storage.BlobTierChanged"
```
Comment (czubair) [inline releases/2021-05/index.md:17]:
```suggestion
- Azure EventGrid SDK for JS has released new GA version. It has added new System Events: `Microsoft.PolicyInsights.PolicyStateCreated`, `Microsoft.PolicyInsights.PolicyStateChanged`, `Microsoft.PolicyInsights.PolicyStateDeleted`, `Microsoft.Storage.AsyncOperationInitiated`, `Microsoft.Storage.BlobTierChanged`.
```

Comment (czubair) [inline releases/2021-05/index.md:17]:
```suggestion
- Azure EventGrid SDK for JS has released new GA version. It has added new Storage System Events: "Microsoft.PolicyInsights.PolicyStateCreated", "Microsoft.PolicyInsights.PolicyStateChanged", "Microsoft.PolicyInsights.PolicyStateDeleted", "Microsoft.Storage.AsyncOperationInitiated", "Microsoft.Storage.BlobTierChanged"
```

File: releases/2021-05/index.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
 
 ## Release Highlights
 
+- SDK for JS has released new low level client packages for these services - Azure Purview Scanning, Azure Purview Catalog, Azure Confidential Ledger and Azure Document Translator
+- SDK for JS has released a new library for Azure Web PubSub service and another library specifically for event handlers for Express
+- Azure Communication Phone Numbers SDK for JS released GA for the first time, with version 1.0.0
+- Azure EventGrid SDK for JS has released new GA version. It has added new System Events: `Microsoft.PolicyInsights.PolicyStateCreated`, `Microsoft.PolicyInsights.PolicyStateChanged`, `Microsoft.PolicyInsights.PolicyStateDeleted`, `Microsoft.Storage.AsyncOperationInitiated`, `Microsoft.Storage.BlobTierChanged`.
```
Comment (KarishmaGhiya) [inline releases/2021-05/index.md:17]:
```suggestion
- Azure EventGrid SDK for JS has released new GA version. It has added new Storage System Events: `Microsoft.PolicyInsights.PolicyStateCreated`, `Microsoft.PolicyInsights.PolicyStateChanged`, `Microsoft.PolicyInsights.PolicyStateDeleted`, `Microsoft.Storage.AsyncOperationInitiated`, `Microsoft.Storage.BlobTierChanged`.
```

File: _data/releases/2021-05/js.yml [hunk 3]
```diff
@@ -72,7 +72,7 @@ entries:
 - Name: '@azure-rest/ai-document-translator'
   Version: 1.0.0-beta.1
   DisplayName: '@azure-rest/ai-document-translator'
-  ServiceName: ""
+  ServiceName: Azure Document Translator
```
Comment (weshaggard) [inline _data/releases/2021-05/js.yml:75]: For future references the service name should not have "Azure" or "Microsoft" as prefix that context is generally implied.

**Pull request merged**